sormas-backend/src/main/resources/sql/sormas_schema.sql

```diff
@@ -10049,4 +10049,12 @@ ALTER TABLE campaigns_history ADD COLUMN archiveundonereason character varying(5
 
 INSERT INTO schema_version (version_number, comment) VALUES (445, 'CoreAdo: Introduce "end of processing date" #7247');
 
+-- 2022-03-04 Configuration for automatic archiving #7775
+ALTER TABLE featureconfiguration ADD COLUMN entitytype character varying(255);
+ALTER TABLE featureconfiguration_history ADD COLUMN entitytype character varying(255);
+
+CREATE TRIGGER
```

Comment:
?

Reply:
wanted to add validations this way but clarified with Martin we don't want them here
so basically forgot to remove it - did it now